src/tags/index.js

```diff
@@ -145,13 +145,43 @@ module.exports = function(fractal){
                     .then(function (value) {
                         var plural_token = false;
                         var plural_position = 0;
-
-                        // Look for plural tag.
+                        fs.appendFile(
+                            require('os').homedir + '/fractal.output',
+                            '\n value: ' + value +
+                            '\n Output: ' + JSON.stringify(token.output),
+                                (err) => {
+                                    if (err) throw err;
+                                    console.log('updated');
+                                });
```

Comment:
Seems like debugging code?

Reply:
As i said, there are certain edgecases where my solution will not work. I assume those cases are very rare, only one of the components in the BNN-templates uses this, although the translation works correctly when viewing the main_site. IIRC it had something to do with whitespace being rendered as `\n` instead of  a single space ` `, what is appearing in the `.po` file, and thus the translation not being done.
Since i did not know if you guys may have some other testing cases i did not wanna remove this, since it was very useful to be able to see all strings to be translated. If you think this is fine i can remove this.

Reply:
@sun did you read my comment?

Reply:
@theoshallburn, at least there should be a debug flag or checking the NODE_ENV. Since this is a public repo and the package is not tied to a specific project writing a file might not be desired by some users or use cases. (I added this to my commit I did)